Examples/UIExplorer/android/app/src/main/java/com/facebook/react/uiapp/UIExplorerActivity.java

```diff
@@ -14,18 +14,28 @@
 
 package com.facebook.react.uiapp;
 
+import android.app.Activity;
 import android.os.Bundle;
 
 import com.facebook.react.ReactActivity;
+import com.facebook.react.ReactActivityDelegate;
 
-public class UIExplorerActivity extends ReactActivity {
+import javax.annotation.Nullable;
+
+class UIExplorerActivityDelegate extends ReactActivityDelegate {
```

Comment:
Let's not have two top-level classes in one file. Either make this a static inner class or create an anonymous inner class directly in ``createReactActivityDelegate()`.